atlassian/bitbucket/base.py

```diff
@@ -1,8 +1,12 @@
 # coding=utf-8
 
+from datetime import datetime
```

Comment:
I think `datetime` should be added as `install_requires` to `setup.py` and to `requirements.txt`.

Reply:
`datetime` is a Python basic lib. https://docs.python.org/3/library/datetime.html